.github/workflows/build-mudlet.yml

```diff
@@ -157,6 +172,54 @@ jobs:
       working-directory: '${{runner.workspace}}/b/ninja'
       run: ctest --output-on-failure
 
+    - name: (Linux/macOS) restore Luarocks for packaging
+      uses: actions/cache@v2
+      with:
+        path: $HOME/.luarocks
+        key: ccache-${{matrix.os}}-${{matrix.compiler}}-${{hashFiles('.github/workflows/build-mudlet.yml')}}
+        restore-keys: ccache-${{matrix.os}}-${{matrix.compiler}}-
+      if: runner.os == 'Linux' || runner.os == 'macOS'
+
+    - name: (Linux) package Mudlet
+      if: runner.os == 'Linux' && matrix.deploy == 'deploy'
+      run: |
+        export PATH="${{env.VCPKG_ROOT}}/installed/x64-linux/tools/lua:$PATH"
+
+        luarocks install --local LuaFileSystem
+        luarocks install --local luautf8
+        luarocks install --local lua-zip
+        luarocks install --local luautf8
+        luarocks install --local LuaSQL-SQLite3 SQLITE_INCDIR="${{env.VCPKG_ROOT}}/installed/x64-linux/include"
+        luarocks install --local lrexlib-pcre
+
+        # workaround https://github.com/lloyd/yajl/issues/209
+        mv ${{env.VCPKG_ROOT}}/installed/x64-linux/lib/libyajl_s.a ${{env.VCPKG_ROOT}}/installed/x64-linux/lib/libyajl.a
+        mv ${{env.VCPKG_ROOT}}/installed/x64-linux/debug/lib/libyajl_s.a ${{env.VCPKG_ROOT}}/installed/x64-linux/debug/lib/libyajl.a
+        # Rock locations search is hardcoded to -L/usr/local/lib and not adjustable
```

Comment:
Technically i believe it is specified in a `config-5.1.lua` file - on my Linux systems that is `/etc/luarocks/config-5.1.lua` and contains:
* Linux:
```lua
rocks_trees = {
   home..[[/.luarocks]],
   [[/usr/local]]
}
```

FTR and by comparison I had a problem with my dual 32/64-bit setup for Mingw-w64 in that the 32-bit one configuration was messed up and I had to solve it so that the 32-bit one was not storing its 32-bit binary modules (the `C` `.dll` ones) in the same place as the 64-bit ones with the same names. By the end I think I was using:

* 32-Bit Windows `C:\msys64\mingw32\etc\luarocks\config-5.1.lua`:
```lua
rocks_trees = {
   { name = [[user]], root = home..[[/.luarocks-x32]] },
   { name = [[system]], root = [[C:/msys64/mingw32]] }
}
```
* 64-Bit Windows `C:\msys64\mingw64\etc\luarocks\config-5.1.lua`:
```lua
rocks_trees = {
   { name = [[user]], root = home..[[/.luarocks-x64]] },
   { name = [[system]], root = [[C:/msys64/mingw64]] }
}
```

Reply:
That seems to be talking about the location of the rocks trees install path, not related to the path the YAJL rock will search for yajl library at.

Reply:
Ah, yeah you may have a point there... sorry for the noise! :man_shrugging: